### PR TITLE
step-74 description

### PR DESCRIPTION
**Pull request by zjiaqi2018**
@tjhei Anything I should add or change?

### CODE REVIEW

**Comment by tjhei** [comment]
Yes, can you please add a new section 2.7 that describes FEIV in a few paragraphs and references the two steps made with it? You can then move the last sentence you wrote into that section as well. Thanks.

**Comment by zjiaqi2018** [comment]
> Yes, can you please add a new section 2.7 that describes FEIV in a few paragraphs and references the two steps made with it? You can then move the last sentence you wrote into that section as well. Thanks.

Added!

**Review by tjhei — APPROVED** [review]
Looks good to me, thank you!
I will wait with merging in case someone else wants to take a look.

File: 9.3/paper.tex [hunk 1]
```diff
@@ -1089,7 +1100,11 @@ \subsection{New and improved tutorials and code gallery programs}
   or alternatively as the second derivative of an energy functional
   that then also yields the residual itself.
 
-\item \texttt{step-74} SIPG for the Poisson equation \todo[inline]{TODO}
+\item \texttt{step-74} implements the symmetric interior penalty Galerkin 
+(SIPG) method for Poisson's equation using the \texttt{FEInterfaceValues} class
+in combination with \texttt{MeshWorker::mesh\_loop()}. This tutorial demonstrates a
+simple and mesh-independent way to assemble cell face integrals.
```
Comment (kronbichler) [inline 9.3/paper.tex:1106]:
```suggestion
simple and mesh-independent way to assemble cell and face integrals.
```
?

File: 9.3/paper.tex [hunk 2]
```diff
@@ -964,6 +965,16 @@ \subsection{Evaluation and integration at arbitrary points}
...
+
+Internally, this class provides an abstraction for two \texttt{FEFaceValues} objects (or \texttt{FESubfaceValues} when using adaptive refinement). The class introduces new interface degrees of freedom indices that are the union of the degrees of freedom indices of the two \texttt{FEFaceValues} objects. The interface degrees of freedom indices can be converted to the corresponding local degrees of freedom indices of the two cells using a helper function. Furthermore, this class supports vector-valued problems in a similar way as \texttt{FEFaceValues} does. We can easily obtain the corresponding scalar or vector values by providing an \texttt{FEValuesExtractors} object. Overall, this class is intended to be a low level replacement for \texttt{MeshWorker} and \texttt{LocalIntegrators}, and a higher level abstraction compared to assembling face terms manually. 
+
+Examples of \texttt{FEInterfaceValues} for scalar problems can be found in \texttt{step-12}, \texttt{step-47}, \texttt{step-50}, and the new tutorial \texttt{step-74}. While examples of vector-valued problems will be added in the future.
```
Comment (kronbichler) [inline 9.3/paper.tex:975]:
```suggestion
Examples of \texttt{FEInterfaceValues} for scalar problems can be found in \texttt{step-12}, \texttt{step-47}, \texttt{step-50}, and the new tutorial \texttt{step-74}. Examples of vector-valued problems will be added in the future. 
```

File: 9.3/paper.tex (hunk 1 above)
Comment (zjiaqi2018) [reply 9.3/paper.tex:1106]: I wanted to say integrals of cell faces, not including cell integrals. Maybe just use face integrals or surface integrals?

Comment (tjhei) [reply 9.3/paper.tex:1106]: Just call it face integrals.

**Review by bangerth — APPROVED** [review]
Thanks. I'll read through it and edit as necessary.

**Pull request merged**